### PR TITLE
logger clean up

### PR DESCRIPTION
**Pull request by burner**
* file is not completely the same on win and posix

### CODE REVIEW

**Comment by joakim-noah** [comment]
In case you still want to know which OS and folder I found the file, it was on linux/x86 and Android/x86, in whichever folder I ran the test runner for this module.

**Comment by burner** [comment]
thanks for the info

File: std/experimental/logger/core.d [hunk 1]
```diff
@@ -1801,6 +1801,13 @@ unittest
     stdThreadLocalLog = new FileLogger("someFile.log");
 
     stdThreadLocalLog = l;
+
+    scope(exit)
+    {
+        import std.file : exists;
```
Comment (schuetzm) [inline std/experimental/logger/core.d]: Did you forget to use `exists`, or did you want to import `remove` instead?

Comment (joakim-noah) [inline std/experimental/logger/core.d]: He was using it in an assert to check the file was removed, but it wasn't working on Windows, so he removed it but forgot to remove the import.

Comment (burner) [inline std/experimental/logger/core.d]: thanks fixed

File: std/experimental/logger/core.d [hunk 2]
```diff
@@ -1801,6 +1801,12 @@ unittest
     stdThreadLocalLog = new FileLogger("someFile.log");
```
Comment (DmitryOlshansky) [inline std/experimental/logger/core.d]: D classic would be to put `scope(exit)` right after stdThreadLocalLog or even before it.Putting scope(exit) at the end looks strange as being `scope(exit)` it makes no difference there.

Also there are some good hints on temporary name in std.file I think.

Comment (burner) [inline std/experimental/logger/core.d]: done

**Comment by burner** [comment]
thanks

**Pull request merged**